src/definitions.md

```diff
@@ -21,3 +21,6 @@ Depending on the files changes and timings, Git will either automatically handle
 ### Pull Request
 A pull request is a feature of GitHub that allows your feature branch to be reviewed before merging into another branch.
 This allows for developers to review each others' code for bugs or missing specifications.
+
```

Comment:
why is there whitespace

Reply:
What is a ZenHub?

Reply:
Zenhub is a great extension for GitHub that allows for us to track projects, issues, etc...
You'll get a taste of it I believe with an Agile development class later on